sdks/java/io/hadoop-input-format/jdk1.8-hifio-tests/src/test/resources/kubernetes/elasticsearch/es_test_data.py

```diff
@@ -0,0 +1,288 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Script to load data on Elasticsearch instance.
+
+#!/usr/bin/python
+
+import json
```

Comment:
This load script adds random data to the ES instance. That means it'll generate new data each time the data load script is run.
However, the IT tests attempt to verify the results from reading the database by hashing the results. 

The plan for performance tests is that each time the test is run they'll:
1. create new instances of the database
2. then run the data load script (this generates new data, which will have a different hash value)
3. then run the ITs (this attempts to validate against a hash value, which must have been checked in)

generating new data each time the data loading script runs means we can't use hashing validation. My recommendation is that you switch to using deterministic methods to generate data. For example: instead of getting a random number each time, just use a for loop with an incrementing value